stablehlo/dialect/StablehloOps.td

```diff
@@ -2707,25 +2709,27 @@ def StableHLO_ScatterOp: StableHLO_Op<"scatter",
        stablehlo.return %0 : tensor<i64>
    }) {
      scatter_dimension_numbers = #stablehlo.scatter<
-       update_window_dims = [2, 3],
-       inserted_window_dims = [0],
-       scatter_dims_to_operand_dims = [1, 0],
-       index_vector_dim = 2>,
+       update_window_dims = [3, 4],
+       inserted_window_dims = [1],
+       input_batching_dims = [0],
+       scatter_indices_batching_dims = [1],
+       scatter_dims_to_operand_dims = [2, 1],
```

Comment:
Let's talk about this change w.r.t. v1.0 -- I'm planning on making the cut on Monday, I think we should make this a post-1.0 change, since I think most users of StableHLO features have integrated with composites (0.19.0) and it's not great to immediately cut v1.0 after a fairly major op is updated

Reply:
SGTM